tests/test_deprecation.py

```diff
@@ -46,6 +46,7 @@ def test_deprecation_warnings_are_accompanied_by_appropriate_comment(
         ), f"Either a 'deprecated-v...' comment or 'DeprecationWarning' call is missing from {file}"
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning:versioneer")
```

Comment:
Because `versioneer` is an external script - we should suppress warnings in our tests that use it. I would modify the script itself but figured it was better to leave it alone.